### PR TITLE
200 release changes

### PR DESCRIPTION
**Pull request by mbcoder**
changing the version and using module system

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,14 +20,17 @@ plugins {
...
 
 repositories {
     mavenCentral()
     maven {
         url 'https://esri.jfrog.io/artifactory/arcgis'
     }
+    maven {
+        url 'https://olympus.esri.com/artifactory/arcgisruntime-repo'
+    }
 }
```
Comment (mbcoder) [inline build.gradle:34]: I will move this before merging

**Review by colinanderson — APPROVED** [review]
@mbcoder looks ok. I would fix the missing new line at the end of file flagged up for `module-info`.

**Comment by colinanderson** [comment]
@mbcoder if you just don't include the lib (remove from Gradle and `module-info`) I think it should still work and you'll get a message like 
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```
The user can then decide how to resolve it. By using nop we are hiding to some extent that you can use an actual logger.

**Comment by mbcoder** [comment]
> See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

Yes fair point about hiding the logger.  I've added a comment to say what we are doing.  I prefer this approach as it shops the app from outputting alarming console output.

**Review by jenmerritt — APPROVED** [review]
Looks fine and tested. 
In build.gradle you can remove this line from the `run` task though (66):
`mainClassName = 'com.mycompany.app.App'`. 
We've removed those from the samples too, as it's defined under `application` now alongside the `mainModule`

**Pull request merged**